packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteFilesControls.tsx

```diff
@@ -25,9 +27,18 @@ export const DeleteFilesControls = ({
     onCancel,
     onStart,
     taskCounts,
```

Comment:
Should this get the same treatment as tableData? I.e. view hook only vends raw tasks and taskCounts are derived at this layer via a function (albeit, a function that is maybe shareable amongst the different views)? I could go either way on this since taskCounts is undifferentiated but it would align more consistently with the view hooks' zones of responsibility

Reply:
I think that makes senese. Mind if I do this in a follow up on the copy view PR?